lua/entities/gmod_wire_expression2/core/custom/tardis2.lua

```diff
@@ -129,79 +133,79 @@ end
 ]]
 
 e2function string entity:tardisGetData(string id)
-    return tostring(this:GetData(id,false))
+    return tostring(self.player, this:GetData(id,false))
```

Comment:
We probably don't want `self.player` on this one

Reply:
I don't actually even know how that happened